civis/io/_files.py

```diff
@@ -235,6 +237,13 @@ def file_to_civis(buf, name, api_key=None, client=None, **kwargs):
     Small or non-seekable file-like objects will be uploaded with a
     single post.
     """
+    if isinstance(buf, str):
+        with open(buf, 'rb') as f:
+            return _file_to_civis(
+                f, name, api_key=api_key, client=client, **kwargs)
```

Comment:
What happens when `buf` is not a `str` here?

Reply:
This should be fixed now.